src/dscanner/main.d

```diff
@@ -361,6 +366,10 @@ Options:
     --recursiveImports <file>
         Similar to "--imports", but lists imports of imports recursively.
 
+    -I <directory>
+        Specify that imported the given directory should be searched for
+        imported modules.
+
```

Comment:
I think this is useful enough to add to README

Reply:
It's already there. https://github.com/dlang-community/D-Scanner#import-listing